src/features/defi/providers/thorchain-savers/components/ThorchainSaversManager/Withdraw/components/Withdraw.tsx

```diff
@@ -253,6 +256,9 @@ export const Withdraw: React.FC<WithdrawProps> = ({ accountId, onNext }) => {
           return Ok(bnOrZero(fastFeeCryptoBaseUnit).toString())
         }
 
+        // Quote errors aren't necessarily user-friendly, we don't want to return them
```

Comment:
Note the heuristics didn't change here, still the same assumptions, which turned out to be correct, see l291 below